R/bind.R

```diff
@@ -0,0 +1,38 @@
+#' Merge several `dm`
+#'
+#' Create a single `dm` from two or more `dm`-objects.
+#'
+#' @param ... `dm`-objects to bind together.
+#' @inheritParams dm_add_tbl
+#'
+#' @details By default table names need to be unique.
+#'
+#' @return `dm` containing the tables and key relations of all `dm`-objects.
+#' @export
+#'
+#' @examples
+#' dm_1 <- dm_nycflights13()
+#' dm_2 <- dm(mtcars, iris)
+#' dm_bind(dm_1, dm_2)
+dm_bind <- function(..., repair = "check_unique", quiet = FALSE) {
+  if (dots_n(...) == 0) return(dm())
```

Comment:
Do we need this special case?

Reply:
Short answer: I am not sure.

You are right, no error is thrown and the printed output of `dm_bind()` without that line looks like that of an empty dm.
Nevertheless there is a difference between `dm()` and the result of `dm_bind()`:
```r
unclass(dm_bind())
> $def
> data frame with 0 columns and 0 rows
dm()
> $def
> # A tibble: 0 x 9
> # … with 9 variables: table <chr>, data <list>, segment <chr>, display <chr>, pks <list<tbl_df[,1]>>, fks  <list<tbl_df[,2]>>, filters <list<tbl_df[,2]>>, zoom <list>,
> #   col_tracker_zoom <list>
```

I would probably keep the special case.

Reply:
Sure -- without a dm we don't get the columns. Let's keep the special case and document it.

Reply:
ok, documented: dab9826